grails-app/services/de/iteratec/osm/util/CsvExportService.groovy

```diff
@@ -15,22 +15,7 @@ class CsvExportService {
      * The {@link org.joda.time.format.DateTimeFormat} used for CSV export and table view.
      */
     static final DateTimeFormatter CSV_TABLE_DATE_TIME_FORMAT = DateTimeFormat.forPattern("dd.MM.yyyy HH:mm:ss")
```

Comment:
We do not need the formatter in the service anymore:

```suggestion
```

PS: Maybe there are some unused imports after deleting this line.

Reply:
I've moved the formatter to the controller where it is actually used.